com.github.Ryujinx.json

```
@@ -0,0 +1,1266 @@
+{
+    "app-id": "com.github.Ryujinx",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
```

Update runtime to 20.08
```suggestion
    "runtime-version": "20.08",
```

---

@LeandroStanger You haven't tested your change, have you? There's no dotnet extension for 20.08 yet. Please don't suggest anything you did not try.

---

Still not dotnet extension